target/linux/ramips/dts/mt7628an_kroks.dtsi

```diff
@@ -0,0 +1,42 @@
+#include "mt7628an.dtsi"
+
+&ethernet {
+	mtd-mac-address = <&factory 0x4>;
+};
+
+&wmac {
+	status = "okay";
+	mediatek,mtd-eeprom = <&factory 0x0000>;
+};
+
+&spi0 {
+	status = "okay";
+
+	flash@0 {
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <25000000>;
+		m25p,fast-read;
```

Comment:
Fast-read is typically only required for 50 MHz+. Does this device run stable with 50 MHz BTW?

Reply:
It looks like it's needed to work around the problem with race condition in SPI driver on MT7628, after firmware update.
I don't know the details nor references to actual problem, though.

Reply:
Might be related to https://github.com/openwrt/openwrt/pull/1743 but I'm completely unaware. @kroks-free 
Is it any harm to keep it with `fast-read`?